app/components/recommended-video.js

```diff
@@ -19,16 +20,28 @@ export default Component.extend(NoScrollMixin, {
 	init() {
 		this._super(...arguments);
 
-		run.later(() => {
-			track({
-				category: 'related-video-module',
-				label: 'reveal-point',
-				action: trackActions.impression,
-			});
+		window.waitForInstantGlobals(function () {
```

Comment:
they are not so instant when you have to wait for them :trollface:

Reply:
🥇